### PR TITLE
Updated image and JavaScript function and created new sections

### PR DESCRIPTION
**Pull request by gracieorozco**
I updated the image to be a picture of myself and the function to reflect a fact on me. I also created new sections, they are: background, projects, LinkedIn and Github, image gallery, what I like to photograph, and hidden talent blog posts. Afterwards, I deployed the project to a live server.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -8,11 +8,224 @@
...
+          calculator before my first class. Afterwards, I worked three hours 
+          on my mobile project finalizing the embedded website and the 
+          additional screen. Then, I went to my classes and began installing 
+          my environment for my software engineering Flask inventory tracking 
+          system project. Exhausted from programming five hours while studying 
+          for classses, I slept early this day in preparation of my work at 4am 
+          the following day. 
+      </p>
       <div id="greeting-container"></div>
+      <p>Click here to get a random message:</p>
```
Comment (KaylaNguyen) [inline portfolio/src/main/webapp/index.html]: You probably want to switch order of these 2 components so the random message shows up after "Click here to get...."

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -16,13 +16,6 @@
  * Adds a random greeting to the page.
  */
 function addRandomGreeting() {
```
Comment (KaylaNguyen) [inline portfolio/src/main/webapp/script.js]: Should I add more greetings so this function actually returns a "random" one?

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -8,11 +8,224 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <h1>Gracie's Portfolio</h1>
```
Comment (KaylaNguyen) [inline portfolio/src/main/webapp/index.html]: Nice job on the portfolio! =)

**Comment by gracieorozco** [comment]
I have fixed the adjustments you mentioned above by changing the placement of the div and adjusted the function name and variables to reflect what it does.

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -13,16 +13,18 @@
...
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+  // Pick a random fact.	
+  const fact = facts[Math.floor(Math.random() * facts.length)];
+  const factContainer = document.getElementById('fact-container');
+  factContainer.innerText = "\n" + fact;
```
Comment (KaylaNguyen) [inline portfolio/src/main/webapp/script.js]: You may not need this. If you want a space between "Click here.." and the message/fact, you should add this to the index.html page instead.

**Pull request merged**